content/en/guides/os-commercial-ecosystem.md

```diff
@@ -0,0 +1,181 @@
+
+* **Legal policies:** The costs for legal resources pertaining to open source projects can be high, but it is also worth keeping in mind that free legal resources exist. The Software Freedom Law Center (SFLC) has a set of very good online resources on how open source licenses and copyrights work, and much more. The SFLC authors are attorneys who were part of creating popular open source licenses and more. Don’t make the mistake of assuming that a free edition of a project needs no legal protections or forethought regarding licensing.
+
+* **Community management:** The community is often the strongest asset a successful open source project can have, and investing resources in growing a healthy community is a best practice. Often, an enthusiastic community grows from a beloved, free edition of a project, and therefore it is a mistake to invest only in community growth for a commercial edition.
+
+* **Access to test infrastructure/environments:** Some open source applications require dedicated servers, specialist tools, automated quality control and/or security hardening. Dedicating resources to a paid and supported edition while starving resources for a free, community edition is a poor strategy, because free community editions often facilitate the first steps that potential customers will take. Ensure open source versions and any open source developers have access to the same tools, checks and automation that your staff developers or paid versions are run on. Parity across different versions and collaborators is key.
+
+* **Invest wisely in education:** open source education is critical to know where and how to interact with the open source ecosystem in a healthy way.
+
+    * Training – Education and training offerings serve a virtuous cycle, where trained users advance the cause of your open source inventions.
+
+    * Certifications – Certification programs help users advertise their skillsets surrounding your inventions and boost their market value and your market value. Remember that you don’t just have to certify people with skillsets, either. Consider The Linux Foundation’s Core Infrastructure Initiative (CII) Badge Program. It includes a Best Practices Badge that can showcase an open source project's commitment to security, fostering trust.
+
+    * Mentorships and internships – Through mentorships and internships, you can foster a wider knowledge base surrounding your open source offerings, and you can create opportunities. Many prospective interns know that open source skills have value in the job market, and you may be able to find interns who are available for very reasonable terms.
```

Comment:
do we really want to promote interns as cheap labor?

Reply:
we should delete the last sentence for sure

Reply:
Thanks,  @xmulligan for the reviews!  :smile:

Reply:
anytime! :)

Reply:
@anajsana are you going to OSPOCon Europe? Would love to chat with you there if you are